docs/usage/templates.md

```diff
@@ -30,3 +30,23 @@ If you want to print pretty JSON with Handlebars you can use the built-in functi
 `{{{stringToPrettyJSON myvar}}}`
 
 In the example above `myvar` is a variable/field, that contains valid JSON.
+
+### encodeURIComponent
+
+If you want to convert a string to a valid URI, use the built-in function `encodeURIComponent` like this:
+
+`{{{encodeURIComponent myvar}}}`
+
+In the example above `myvar` is the string you want to transform into a valid URI.
+
+Read the [MDN Web Docs, encodeURIComponent()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) to learn more.
+
+### replace
+
+If you want to replace some characters in a string, use the built-in function `replace` like this:
+
+`{{{replace find, replace, context}}}`
+
+In the example above ...
```

Comment:
This needs to be expanded/corrected by somebody who knows how this works. 😉

Reply:
Wondering if these variable names can be renamed to be more understandable

Reply:
I copied this from our code:

https://github.com/renovatebot/renovate/blob/98e702909011d2079d6d50eab372eb6d5b3e38f2/lib/util/template/index.ts#L9-L13

I don't know if my example in the documentation works.

I agree the variable names should be changed to something better! 😄

Reply:
Yeah, `context` is the `string` on which the replace should be done. `replace` is the new value for all occurences of `find` value.

Reply:
Let's use some "real" examples for each

Reply:
I could really use some help to finish the documentation. Can any of you guys provide a suggestion that I can apply to this PR?